lib/evm_database_ops.rb

```diff
@@ -115,14 +116,18 @@ def self.restore(db_opts, connect_opts = {})
         uri = File.join(connect_opts[:uri], backup_folder, connect_opts[:remote_file_name])
       end
 
+      connect_opts[:username] = db_opts[:username] if connect_opts[:username].nil?
+      connect_opts[:password] = db_opts[:password] if connect_opts[:password].nil?
```

Comment:
Why would we want to assume the connection options to the s3 store would ever be the same as the database authentication values?